src/parser/Parser.rl

```diff
@@ -237,6 +237,7 @@ unichar readUtf8CodePoint2c(const char *s) {
 
 static
 unichar readUtf8CodePoint3c(const char *s) {
+    // cppcheck-suppress cstyleCast
```

Comment:
since this generates a C++ file, so a reinterpret_cast should be used instead of a suppress.